web-client/src/components/SignUpForm.jsx

```diff
@@ -0,0 +1,354 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Checkbox from "@material-ui/core/Checkbox";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Button from "@material-ui/core/Button";
+import OutlinedInput from "@material-ui/core/OutlinedInput";
+import InputLabel from "@material-ui/core/InputLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import Container from "@material-ui/core/Container";
+import Visibility from "@material-ui/icons/Visibility";
+import VisibilityOff from "@material-ui/icons/VisibilityOff";
+import IconButton from "@material-ui/core/IconButton";
+import InputAdornment from "@material-ui/core/InputAdornment";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import * as yup from "yup";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+
+import "../App.css";
+
+const styles = makeStyles({
+  main: {
+    padding: "2%",
+    display: "flex",
+  },
+  form: {
+    display: "flex",
+    flexDirection: "column",
+    float: "left",
+    textAlign: "left",
+    maxWidth: "400px",
+    margin: "16px",
+  },
+  textField: {
+    marginBottom: "32px",
+    width: "300px",
+  },
+  formControl: {
+    marginBottom: "32px",
+    width: "300px",
+  },
+  formControlLabel: {
+    marginBottom: "8px",
+  },
+  btnSignup: {
+    width: "100px",
+  },
+  menuItem: {
+    textAlign: "left",
+  },
+  inputLabel: {
+    backgroundColor: "white",
+    padding: "0px 4px",
+    marginLeft: "-4px",
```

Comment:
negative value? :O

Reply:
unfortunately I have to because InputLabel and OutlinedInput does not provide white background for the label it won't be aligned
![image](https://user-images.githubusercontent.com/44305068/102511501-2592b180-4089-11eb-8bc7-ec9968d59a3c.png)

Reply:
nvm then